kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README

```diff
@@ -0,0 +1,55 @@
+This sample has been enhanced to provide the ability for a user to introspect an already existing domain
+(e.g. on-prem domain) using the WebLogic Deploy Tooling (WDT) and then create the same domain on Persistent Volume (PV)
+in Kubernetes using this sample. The WDT Discover tool will generate a WDT model yaml (metadata representation of the
+domain configuration) and a properties file which will contain a key=value for each property in the WDT model.
+The sample has been enhanced to update the domain using both WDT Update and the WDT Extract Domain Resource Tools.
+
+New functionality for the Domain Home on PV sample:
+1- The create domain script create-domain.sh will use the WDT Extract Domain Resource Tool to generate the domain CR yaml (domain.yaml).
+2- The ability to update the domain on PV which was originally created as described above using WDT. The domain
+   configuration will be updated by running WDT Update Tool using a WDT full model and the equivalent change to
+   the Domain CR using the WDT Extract Domain Resource Tool.
+3- The create and update domain will accept the following inputs: a WDT model, and the WDT properties file
+    (e.g. generated during WDT discovery).
+
+Note: You can still create the domain by updating the create-domain-inputs yaml file.
+The updateDomain script update-domain.sh only supports updating a domain created using the WebLogic Deploy Tooling.
+
+Use case - WDT can be used to discover an domain (could be on-prem). Running WDT Discover Tool
+generates a WDT model file and a property file. When introspecting an existing domain with WDT the resulting model
+might need to be modified for compatibility with specific target environments, such as Oracle Weblogic Server
+Kubernetes Operator. The user should also update the file create-domain-inputs.yaml for operator specific inputs.
+
+This sample can take both the input yaml file and the generated properties file as inputs to 
+createDomain.sh and updateDomain.sh as a comma separated list.
+
+To create a domain using this sample -
+
+   1. Invoke create-domain.sh -i  /path/to/directory/create-domain-inputs.yaml,/path/to/directory/properties-file
+      -m /path/to/directory/users-wdt-model -o outputdir -v
+         The above command will create a domain on PV and also a domain.yaml file in outputdir/weblogic-domain/domainname
+         The order of the input files does not matter.
+   2. The domain.yaml generated by the WDT Extract Domain Resource Tool will have '--FIX ME--' statements for
+      those values it can not generate. The user is expected to fill in the information identified by --FIX ME--
+      in the domain resource output. Make sure the domain.yaml does not contain any '--FIX ME--' before proceeding
```

Comment:
domain.yaml does not contain any '--FIX ME--'  -> `domain.yaml` file does not contain any '--FIX ME--' statements